h/groups/__init__.py

```diff
@@ -1,4 +1,8 @@
 # -*- coding: utf-8 -*-
 
+from h.groups import search
+
+
 def includeme(config):
     config.memex_set_groupfinder('h.groups.util.fetch_group')
+    config.memex_add_search_filter(search.GroupAuthFilter)
```

Comment:
I'd suggest we unit test this the same way `pyramid_services` suggests you test service registration: https://github.com/mmerickel/pyramid_services#testing-examples.

Specifically: add a request method and configuration directive `memex_get_search_filters`, and then test the includeme explicitly:

```python
def test_includeme_adds_groupauthfilter(pyramid_config):
    pyramid_config.include('h.groups')
    
    assert search.GroupFilter in pyramid_config.memex_get_search_filters()
```

(And then we can use that (in request method form) in `memex.search.core.default_querybuilder`, too.)